bigbluebutton-html5/imports/ui/components/whiteboard/service.js

```diff
@@ -7,22 +7,23 @@ import { isEqual } from 'lodash';
 import WhiteboardToolbarService from './whiteboard-toolbar/service';
 
 const Annotations = new Mongo.Collection(null);
-let _isDiscarded = false;
+let isDiscarded = false;
 
 function clearFakeAnnotations() {
   Annotations.remove({ id: /-fake/g });
 }
 
 function discardCurrentAnnotation(whiteboardId) {
-  if (_isDiscarded === true) {
-    WhiteboardToolbarService.undoAnnotation(whiteboardId);
-    _isDiscarded = false;
-  }
+  WhiteboardToolbarService.undoAnnotation(whiteboardId);
+  isDiscarded = false;
 }
 
 function handleAddedAnnotation({
   meetingId, whiteboardId, userId, annotation,
 }) {
+  const drawStartStatus = Meteor.settings.public.whiteboard.annotations.status.start;
```

Comment:
you can add a constant `annotationStatus` and declare `drawStartStatus = annotationStatus.start`. As it is too long lines

Reply:
and you can declare a above scope and avoid re-declare same variable